docs/commands/init.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Init Command
+description: Learn more about the `init` command in Melos.
+---
+
+# Init Command
+
+The `init` command initializes a new Melos workspace. It creates the necessary configuration files and directory structure for your monorepo.
+
+## Basic Usage
+
+```bash
+melos init [workspace_name]
+```
+
+If no workspace name is provided, you'll be prompted to enter one. By default, it uses the current directory name.
+
+## Options
+
+### --directory (-d)
+Specifies the directory where the workspace should be created. If not provided, you'll be prompted to enter one. Defaults to the workspace name, or current directory ('.') if the workspace name matches the current directory name.
```

Comment:
I think we have a max line length of 80 or 100 on the other markdown files if I remember correctly. (We should really add a markdown linter to the pipeline)

Reply:
Updated to <80 chars